.github/workflows/spectral-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Spectral Lint Commited OpenAPI Spec
+
+# Trigger the workflow on pull requests and pushes to the main branch
+on:
+  pull_request:
+    paths: 
+      - 'tools/spectral/**'
+      - 'openapi/**.yaml'
```

Comment:
```suggestion
      - 'tools/spectral/**'
      - 'openapi/**.yaml'
      - '.spectral.yaml'
```

Reply:
is this valid `- '.spectral.yaml'`? I'm moving .spectral.yaml to tools/spectral

Reply:
ah sorry, I missed this change. You need to update the required-validation workflow to make sure the command to run spectral uses the file in the new location